integration/e2e/images/images.go

```diff
@@ -9,6 +9,7 @@ var (
 	Memcached        = "memcached:1.6.1"
 	Minio            = "minio/minio:RELEASE.2019-12-30T05-45-39Z"
 	Consul           = "consul:0.9"
+	ETCD             = "gcr.io/etcd-development/etcd:v3.4.7"
```

Comment:
As mentioned on the top of this file, please add this image to the preloading done in `.circleci/config.yml`